backend/test/org/akvo/dash/fixtures.clj

```diff
@@ -1,21 +1,43 @@
 (ns org.akvo.dash.fixtures
   (:require
    ;; [duct.component.ragtime :as ragtime]
-   [reloaded.repl :refer [system stop go]]))
+   [hugsql.core :as hugsql]
+   [meta-merge.core :refer [meta-merge]]
+   [org.akvo.dash.migrate :as migrate]
+   [org.akvo.dash.component.tenant-manager :refer [connection]]
+   [org.akvo.dash.config :as config]
+   [reloaded.repl :refer [system stop go]]
+   [ring.middleware.stacktrace :refer [wrap-stacktrace]]))
 
+(hugsql/def-db-fns "org/akvo/dash/fixtures.sql")
 
-(defn system-fixture
-  "Just a dummy fixture while we rework stuff"
-  [f]
-  (f))
+(def dev-config
+  {:app {:middleware [wrap-stacktrace]}})
+
+(def config
+  (meta-merge config/defaults
+              config/environ
+              dev-config))
+;(defn system-fixture
+;  "Just a dummy fixture while we rework stuff"
+;  [f]
+;  (f))
 
 
-;; (defn system-fixture
-;;   "Starts the system and migrates, no setup or tear down."
-;;   [f]
-;;   (try
-;;     (go)
-;;     (-> system :ragtime ragtime/reload ragtime/migrate)
-;;     (f)
-;;     (-> system :ragtime ragtime/reload (ragtime/rollback "001-activity"))
-;;     (finally (stop))))
+(defn system-fixture
+  "Starts the system and migrates, no setup or tear down."
+  [f]
+  (let [conn {:connection-uri (-> config :db :uri)}]
+    (try
+      (go)
+      (migrate/migrate conn)
+      (insert-tenant conn {:db_uri "jdbc:postgresql://localhost/test_dash_tenant_1?user=dash&password=password"
```

Comment:
@zzgvh I know you talked about refactoring this two inserts to a function, but for me I think this is all fine.